one-ui/src/main/java/com/marklogic/hub/oneui/controllers/EnvironmentController.java

```diff
@@ -122,4 +127,18 @@ public JsonNode reset() {
         obj.put("reset", true);
         return obj;
     }
+
+    @RequestMapping(value = "/project-info", method = RequestMethod.GET)
+    @ResponseBody
+    public JsonNode getProjectInfo(HttpSession session) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectNode resp = objectMapper.createObjectNode();
+        resp.put("dhfVersion", versions.getHubVersion());
+        String projDir = hubConfig.getHubProject().getProjectDir().toString();
+        projDir = projDir.substring(projDir.lastIndexOf(File.separator)+1);
```

Comment:
This logic is repeated above. Seems like HubProject should have a "getProjectName()" method, assuming that project name should always equal the name of the directory? 

Also, since getProjectDir returns a Path, it seems better to call toFile().getName() instead of messing around with Strings.